### ISSUE TITLE
installation from libTWOLAME (pacman) installed also FLAC, speex and SND-file

### ISSUE THREAD

**Issue by pat357** [jb-alvarado/media-autobuild_suite#169]
As the titel says it seems that flac, speex and snd-file are installed as deps  if you install twolame.

Because we also compile flac, speex and snd-file, this will almost certain lead to problems when compiling these.
The mingw versions may have different include  headers and libs which are build with different options and even different versions which can confuse our own compilations.

I added the following to my script to avoid these problems:
For compilation of speex:
    rm -rf $MINGW_PREFIX/include/speex
    rm -f $MINGW_PREFIX/lib/libspeex{,dsp}.{dll,}a{,.dyn} $MINGW_PREFIX/lib/pkgconfig/speex{,dsp}.pc

For compilation of FLAC:
    rm -rf $MINGW_PREFIX/include/FLAC{,++}
    rm -f $MINGW_PREFIX/lib/libFLAC{,++}.{dll,}a{,.dyn}  $MINGW_PREFIX/lib/pkgconfig/flac{,++}.pc

For compilation of SND-file:
     rm -rf $MINGW_PREFIX/include/sndfile.{h,}h
     rm -f $MINGW_PREFIX/lib/libsndfile.{dll,}a{,.dyn}  $MINGW_PREFIX/lib/pkgconfig/sndfile.pc

BTW : you seems to use rm -rf  to delete the own previous compilation from these specific libs, even when their are no dirs involved. (rm -f  should be enough, but I'm sure you knew that). Just a typo, I guess.

What's your idea about this ? 
What would be the best approach in your opinion ?
Because flac and speex are not from git (no automatic updates), it's maybe interesting to replace the own compilations by the Mingw version ? (only if these mingw builds are usable/compatible for ffmpeg/mplayer/mpv,...obviously)

**Comment by wiiaboo** [comment]
Both flac and speex are compiled because people want the standalone encoder for it.
And the versions are exactly the same in the script and mingw.
The patch in speex is exactly the same as the one mingw uses since it was stolen from there and flac isn't used anywhere important, besides also being the same version.
Unless there's some problems compiling other stuff it shouldn't matter if mingw's versions are also installed.
Flac isn't used in ffmpeg since it has its own native encoder and decoder. It's just used in sndfile.

tl;dr: probably safe to ignore until issues arise

**Comment by wiiaboo** [comment]
Also, you don't need to remove *.dyn files since they aren't used anywhere, they're just *.dll.a renamed as backups to not be used accidentaly.
